stdlib/array.gr

```diff
@@ -92,6 +92,33 @@ export let mapi = (fn, array) => {
   })
 }
 
+export let fill = (value, array) => {
+  let length = length(array)
+  forEachi((_, index) => {
+    array[index] := value
+  }, array)
+  void
```

Comment:
We can drop the void now!
```suggestion
```

Reply:
Oh, actually the tests failed when I removed void, haha.

Reply:
Oh, dang.